mmv1/templates/terraform/examples/backup_dr_backup_plan.tf.tmpl

```diff
@@ -0,0 +1,160 @@
+variable "backup_rules" {
+  type = list(object({
+    rule_id                   = string
+    backup_retention_days     = number
+    recurrence_type           = string
+    frequency                 = number
+    time_zone                 = string
+    start_hour_of_day         = number
+    end_hour_of_day           = number
+    days_of_week              = list(string)  # Optional for WEEKLY
+    days_of_month             = list(number)  # Optional for MONTHLY
+    week_day_of_month         = object({      # Optional for MONTHLY
+      week_of_month = string
+      day_of_week   = string
+    })
+    months                    = list(string)  # Optional for YEARLY
+  }))
+}
+default = [
```

Comment:
We don't normally see testing like this, and I believe it is just because it is harder to isolate why the test failed vs having separate examples. These also shows up in the docs, and for acceptance tests, we are normally more interested in the API interaction vs the full behavior of the endpoint. So that's all to say, I would maybe simplify this into 2 separate examples: 1 with a minimal config, and another with all fields specified.